harvard-lts/FITSservlet
# LTSMAINT-329 - Update to log4j2

**JIRA Ticket**: ([link](https://jira.huit.harvard.edu/browse/LTSMAINT-329))

# What does this Pull Request do?
Updates log4j1 to log4j2

# How should this be tested?
`mvn clean install`
Deploy the created WAR file into Tomcat 8.5.x.
It will not fully startup since the server won't be configured to find the full FITS deployment. However, in catalina.out there should be a reference to log4j2.xml and there should be the file `fits-service.log` created in the Tomcat `logs` directory.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no

# Interested parties
@awoods @cvicary @jessjass @michael-lts
